plugins/ru.runa.gpd/plugin.properties

```
@@ -1141,6 +1141,8 @@ RenameBotDialog.update.title = Rename bot
 RenameBotDialog.error.botWithSameNameExists=Bot with name "{0}" already exists
 RenameBotTaskDialog.update.title = Rename bot task
 
+RenameTransitionDialog.update.title = Rename Transition object
```

```suggestion
RenameTransitionDialog.update.title = Rename transition
```

Думаю, так достаточно. И соответственно русский перевод - Переименовать переход, если он сейчас не такой